src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Do you really need all these fonts?

Reply:
> Do you really need all these fonts?

Nope, I just copied all the Roboto fonts because I didn't know which ones I'd use in the project. Is it better to only take the fonts that I'll use in the project?